app/build.gradle

```diff
@@ -3,18 +3,22 @@ plugins {
 }
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+androidExtensions {
+    experimental = true
+}
 
 android {
-    compileSdkVersion 27
+    compileSdkVersion 28
     buildToolsVersion '28.0.3'
 
     defaultConfig {
         applicationId 'com.nilhcem.blenamebadge'
-        minSdkVersion 18
-        targetSdkVersion 27
+        minSdkVersion 21
```

Comment:
Why?

Reply:
most of the devices are lollipop+. should I revert it back to 18 ?

Reply:
Because we don't use any API that is only available in API 21+, why deliberately increase minSdk when there is no need?